microsoft/PowerToys
# Window styles tool

## Summary of the Pull Request

**What is this about:**

Tool for getting styles of the window to check if FancyZones can snap it to a zone. 

![image](https://user-images.githubusercontent.com/8949536/164212392-b6b5a8bf-8670-4815-813b-4f01a56947a5.png)

**What is included in the PR:** 

**How does someone test / validate:** 

Build and run the project, select a window and press a shortcut. Make sure that the output file was created on a desktop and tool's window was closed. If you run the tool several times, new info will be appended to the file.

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
